### PR TITLE
Implement applyTo method for TensorAddUpdate.

### PR DESCRIPTION
**Pull request by toregge**
Implement print method for TensorAddUpdate and TensorModifyUpdate.

@geirst :  please review

### CODE REVIEW

File: document/src/vespa/document/update/tensormodifyupdate.cpp [hunk 1]
```diff
@@ -50,6 +50,23 @@ getJoinFunction(TensorModifyUpdate::Operation operation)
     }
 }
 
+const char *
```
Comment (geirst) [inline document/src/vespa/document/update/tensormodifyupdate.cpp]: Consider returning vespalib::string instead.

**Review by geirst — APPROVED** [review]
👍

File: document/src/vespa/document/update/tensormodifyupdate.cpp (hunk 1 above)
Comment (toregge) [reply document/src/vespa/document/update/tensormodifyupdate.cpp]: Done

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**